### ISSUE TITLE
Pass reserved=TRUE to URLEncode

### ISSUE THREAD

**Issue by isteves** [NCEAS/rt#17]
As per Bryce's comment in #15

**Comment by isteves** [comment]
@amoeba tried this change on rt_ticket_search but it seemed to break it when I ran `rt_ticket_search(query = "Queue='General'AND(Status='new')")`

 ```
Error in curl::curl_fetch_memory(url, handle = handle) : 
  Could not resolve host: https%3A%2F%2Fdemo.bestpractical.com%2FREST%2F1.0%2Fsearch%2Fticket%3Fquery%3DQueue%3D%27General%27AND%28Status%3D%27new%27%29%26format%3Dl 
```

**Comment by amoeba** [comment]
Looks like you URLencoded the whole URL and not just the query parameter. If that's the case, just URLencode `'General'AND(Status='new')`

**Comment by isteves** [comment]
fixed here 857b247b9b8df92b17db228f7fd8dbf3c5ce0628

**Comment by amoeba** [comment]
🎉
